driver/mobile/device.go

```diff
@@ -2,6 +2,10 @@ package mobile
 
 // Device describes functionality only available on mobile
 type Device interface {
-	ShowVirtualKeyboard() // Request that the mobile device show the touch screen keyboard (standard layout)
-	HideVirtualKeyboard() // Request that the mobile device dismiss the touch screen keyboard
+	// Request that the mobile device show the touch screen keyboard (standard layout)
+	ShowVirtualKeyboard()
+	// Request that the mobile device show the touch screen keyboard (custom layout)
+	ShowVirtualKeyboardType(Keyboard)
```

Comment:
Do we need new method or we can break compatibility? Seems pretty minor for me. Also can't think of a better name than what's already in use.

Reply:
You're right we need a new method here.
If this is deemed ugly we could change in 2.0 work.

Reply:
Not sure. We can introduce `ShowVirtualKeyboardType` or something in upcoming release and remove in 2.0. But if nobody needs it, we can wait until 2.0.